Bottom: fd530ae5cf369087b768bf7fbe403d75ce8c5a2a
Top:    235442bd7252d876c4a7ab085c8d428777f05fe9
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-29 15:25:46 -0500

Refresh of buildbot-worker-ntel

---
diff --git a/docker/buildbot-worker-ntel/buildbot/buildbot.tac b/docker/buildbot-worker-ntel/buildbot/buildbot.tac
index 472f609e..058c6a08 100644
--- a/docker/buildbot-worker-ntel/buildbot/buildbot.tac
+++ b/docker/buildbot-worker-ntel/buildbot/buildbot.tac
@@ -17,7 +17,7 @@ application.setComponent(ILogObserver, FileLogObserver(sys.stdout).emit)
 # and worker on the same process!
 buildmaster_host = os.environ.get("BUILDBOT_MASTER", 'localhost')
 port = int(os.environ.get("BUILDBOT_WORKER_PORT", 9989))
-workername = os.environ.get("BUILDBOT_WORKER_NTEL", 'docker')
+workername = os.environ.get("BUILDBOT_WORKER_NAME", 'docker')
 passwd = os.environ.get("BUILDBOT_WORKER_PASSWORD")
 
 # delete the password from the environ so that it is not leaked in the log
